.gitignore

```diff
@@ -5,6 +5,8 @@ CNAME
 .travis.yml
 .idea/
 .vscode/
+tmp/
+bin/htmltest
```

Comment:
This might not be needed if `htmltest` runs in a container.

Reply:
:+1: